sql/pom.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.stargate</groupId>
+    <artifactId>stargate</artifactId>
+    <version>0.0.16-SNAPSHOT</version>
+  </parent>
+  <groupId>io.stargate.api.sql</groupId>
+  <artifactId>sql-base</artifactId>
+  <packaging>pom</packaging>
+  <properties>
+    <junit.version>5.6.2</junit.version>
```

Comment:
You don't want `<junit.version>5.7.0</junit.version>` from the root pom?

Reply:
Missed that on copy-paste :)

Reply:
fixed in the latest push